src/main/scala/is/hail/methods/PCRelate.scala

```diff
@@ -29,34 +33,173 @@ object PCRelate {
     def map[N](f: M => N): Result[N] = Result(f(phiHat), f(k0), f(k1), f(k2))
   }
 
-  val defaultMinKinship = Double.NegativeInfinity
+  val defaultMinKinship: Double = Double.NegativeInfinity
   val defaultStatisticSubset: StatisticSubset = PhiK2K0K1
+  val defaultStorageLevel: StorageLevel = StorageLevel.MEMORY_AND_DISK
+
+  private[this] def apply(
```

Comment:
there are 7 overloaded methods here. Can we reduce this to 2 (one for Scala and one for Python)?